src/erlcloud_kms.erl

```diff
@@ -114,16 +114,16 @@ default_config() ->
 %%
 %%------------------------------------------------------------------------------
 -spec create_alias
-          (AliasName :: string(),
-           TargetKeyId :: string()) ->
+          (AliasName :: binary(),
```

Comment:
@kkuzmin @tomdos 
speaking of whether we should have `string()|binary()` here, I disagree with the idea of having two options, why we would need that at all?

`erlcloud_kms` gives you binaries on output (as it uses `jsx:decode`) and ask for binaries, which sounds logical and consistent. Having separate `string()` input sounds like something out of the place to me.